config/development.config.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
Shouldn't this file be excluded via `.gitignore`? It will be copied from `development.config.php.dist` when in development mode...

Reply:
Good point, fixed in 8c5058535c5d10ff9fea2050a18c59e6f8106f66